docs/src/assets/scss/components/docs-navigation.scss

```diff
@@ -71,8 +78,32 @@
     }
 }
 
-.docs-nav-panel .mobile-only {
-    @media all and (min-width: 1023px) {
+#nav-panel{
+    @media all and (max-width: 1023px) {
+        display: none;
+    }
+}
+
+#nav-panel[data-open="true"]{
+    @media all and (max-width: 1023px) {
+    display: block;
+    }
+}
+
+#js-docs-index-list {
+    @media all and (max-width: 1023px) {
+        display: none;
+    }
+}
+
+#js-docs-index-list[data-open="true"]{
+    @media all and (max-width: 1023px) {
+    display: block;
+    }
+}
```

Comment:
small nit: We could have extracted the media query outside and could have had all the selectors + attributes + values inside the same media query?  I see that all media queries are common?